src/main/java/cloud/cholewa/game/components/PasswordManager.java

```diff
@@ -0,0 +1,26 @@
+package cloud.cholewa.game.components;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+public class PasswordManager {
+    private final List<String> passwords = new ArrayList<>();
+
+    public PasswordManager() {
+        passwords.addAll(List.of(
```

Comment:
Jest ok, natomiast można odrobinę poprawić. w 8. linii nie robić nowej ArrayList, a w konsturktorze od razu dać
 passwords = List.of (....)

Reply:
done